content/en/containers/guide/autodiscovery-management.md

```diff
@@ -13,17 +13,26 @@ further_reading:
 
 By default, the Datadog Agent automatically discovers all containers available. To restrict its discovery perimeter and limit data collection to a subset of containers only, include or exclude them through a dedicated configuration.
 
-**Note**: The `kubernetes.containers.running`, `kubernetes.pods.running`, `docker.containers.running`, `.stopped`, `.running.total`, and `.stopped.total` metrics are not affected by these settings and always count all containers.
+## Autodiscovery patterns
 
-## Environment variables
-If you are running the Agent as a container, use the [Containerized Agent](?tab=containerizedagent) tab instructions. If you are running the Agent as a binary on a host, use the [Host Agent](?tab=hostagent) tab instructions. 
+The Datadog Agent should be deployed once per host within containerized environments. This is typically done with a DaemonSet in Kubernetes (managed by Helm or Operator) or by ECS Daemon Services. Each Datadog Agent deployed automatically discovers and monitors all containers on its respective host.
 
-{{< tabs >}}
-{{% tab "Containerized Agent" %}}
+You can adjust the discovery rules for the Agent to restrict metric and log collection. Any containers restricted from metric collection are also restricted for any Autodiscovery based Agent Integrations. When the [log "Container Collect All" feature][1] is enabled all discovered containers have their logs collected, unless otherwise blocked by the rules described below.
```

Comment:
```suggestion
You can adjust the discovery rules for the Agent to restrict metric and log collection. Any containers restricted from metric collection are also restricted for any Autodiscovery based Agent Integrations. When the [log "Container Collect All" feature][1] is enabled, all discovered containers have their logs collected, unless otherwise blocked by the rules described below.
```